accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -539,11 +544,65 @@ public static void main(String[] args)
         }
     }
 
-    @Test
-    @Timeout(value = 3, unit = TimeUnit.MINUTES)
+//    @Test
+    @RepeatedTest(Integer.MAX_VALUE)
     public void testOne()
     {
-        run(1L, 1000);
+        run(System.nanoTime());
+//        run(320860822860166L); // Incomplete txn ({read:[]}) provided; does not cover [(542#7281,542#10920]] // In debugging the txn was a historic txn
+//        run(321378044901625L); // Incomplete txn ({read:[]}) provided; does not cover [(849#4681,849#9362], (849#9362,849#10854]]
+//        run(321814000189583L); // Incomplete txn ({read:[]}) provided; does not cover [(36#13106,36#14320]]
+
+//        run(321402699680416L); // Inconsistent execution timestamp detected for command Command@1486670059{[5,26002,6(KS),15]:Committed}: [5,26002,6(KS),15] != [5,26031,111,9]
+//        run(321824911459041L); // Inconsistent execution timestamp detected for command Command@45621272{[3,21004,6(KS),7]:PreApplied}: [3,21004,6(KS),7] != [3,21817,10,12]
+
+//        run(321453140939208L); // ISE in accord.local.SerializerSupport.truncated(SerializerSupport.java:171)
+//        run(325079839240208L); // ISE in accord.local.SerializerSupport.truncated(SerializerSupport.java:171)
+
+//        run(325076065173541L); // Invalid state: waiting for execution of command that is not owned at the execution time
+
+//        run(325459494008416L); // Writes is null
+    }
+
+    private static void run(long seed)
```

Comment:
I never use `main` but could migrate this to include the operation so `main` has the same timeout logic.  I pulled this from `Property` class as its used to avoid loosing the seed when tests take longer than expected...